__tests__/user-details/task-duedate-hover.test.js

```diff
@@ -48,11 +40,11 @@ describe('Tasks On User Management Page', () => {
             'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS',
             'Access-Control-Allow-Headers': 'Content-Type, Authorization',
           },
-          body: JSON.stringify(superUserDetails), // Y contains the json of a superuser in the server which will grant us the access to view the page without locks
+          body: JSON.stringify(userDetails),
         });
       } else if (
-        url ===
-        'https://api.realdevsquad.com/tasks/?size=3&dev=true&assignee=ajeyakrishna'
+        url === `${API_BASE_URL}/users/self` ||
```

Comment:
This is deprecated, please update it

Reply:
I can create a separate PR for fixing this deprecated route for this whole repo.